pkg/iac/scanners/terraform/module_test.go

```diff
@@ -567,7 +567,7 @@ resource "something" "else" {
 		for_each = toset(["true"])
 
 		content {
-			ok = each.value
+			ok = blah.value
```

Comment:
For reviewers: this test should not have run successfully because dynamic blocks do not create an `each` object.